testament/testament.nim

```diff
@@ -86,6 +93,10 @@ var targetsSet = false
 
 proc isSuccess(input: string): bool =
   # not clear how to do the equivalent of pkg/regex's: re"FOO(.*?)BAR" in pegs
+  # note: this doesn't handle colors, eg: `\e[1m\e[0m\e[32mHint:`; while we
+  # could handle colors, there would be other issues such as handling other flags
+  # that may appear in user config (eg: `--listFullPaths`).
+  # Passing`XDG_CONFIG_HOME= ` is one way to avoid interefence from user config.
```

Comment:
```suggestion
  # Passing `XDG_CONFIG_HOME= ` is one way to avoid interefence from user config.
```

Reply:
Please give notes or link for `XDG_CONFIG_HOME= `

Reply:
fixed